modules/table/src/test/java/org/apache/ignite/internal/table/RecordBinaryViewOperationsTest.java

```diff
@@ -343,6 +343,24 @@ public void getAll() {
         assertThat(res, contains(rec1, null, rec3));
     }
 
+    @Test
+    public void testContains() {
+        SchemaDescriptor schema = schemaDescriptor();
+        RecordView<Tuple> tbl = createTable(schema).recordView();
+
+        final long keyId = 1L;
+        Tuple rec = Tuple.create()
+                .set("id", keyId)
+                .set("val", 11L);
+        Tuple keyRec = Tuple.create()
+                .set("id", keyId);
+
+        tbl.insert(null, rec);
+
+        assertTrue(tbl.contains(null, keyRec));
```

Comment:
Let's add tests for cases:
1. `null` value can be found
`tbl.insert(null, Tuple.create().set("id", 1L).set("val", null);`
`assertTrue(tbl.contains(null, keyRec));`
2. removed row can NOT be found
`tbl.delete(null, keyRec);`
`assertFalse(tbl.contains(null, keyRec));`

Reply:
Done~